src/frequenz/client/microgrid/_component_data.py

```diff
@@ -363,18 +387,30 @@ class EVChargerData(ComponentData):  # pylint: disable=too-many-instance-attribu
     """A wrapper class for holding ev_charger data."""
 
     active_power: float
-    """The 3-phase active power, in Watts, represented in the passive sign convention.
-        +ve current means consumption, away from the grid.
-        -ve current means supply into the grid.
+    """The 3-phase active power, in Watts (W).
```

Comment:
Maybe it's better to be explicit here?
```suggestion
    """The total 3-phase active power, in W (Watt).
```